pkg/kubernetes/helper.go

```diff
@@ -395,3 +397,18 @@ func EnsureLabels(o metav1.Object, toEnsure map[string]string) {
 	}
 	o.SetLabels(labels)
 }
+
+type SeedClientMap map[string]ctrlruntimeclient.Client
+
+type SeedVisitorFunc func(seedName string, seedClient ctrlruntimeclient.Client, log *zap.SugaredLogger) error
```

Comment:
is the seedName used anywhere? I see SeedVisitorFunc used like
`r.seedClients.Each(ctx, log, func(_ string, seedClient ctrlruntimeclient.Client, log *zap.SugaredLogger)`  everywhere. But maybe i am missing it?

Reply:
Not yet, but I can imagine the name being useful and I then do not want to refactor the entire codebase again. 😁

Reply:
ok makes sense